tests/integrationtest/lib.go

```diff
@@ -922,25 +922,30 @@ func NormalControllerServer(w http.ResponseWriter, r *http.Request, args ...stri
 	addrPrefix := "10.250.250"
 	object := strings.Split(strings.Trim(url, "/"), "/")
 
-	if strings.Contains(url, "macro") && r.Method == "POST" {
+	if r.Method == "POST" && !strings.Contains(url, "login") {
 		data, _ := ioutil.ReadAll(r.Body)
 		json.Unmarshal(data, &resp)
-		rData, rModelName := resp["data"].(map[string]interface{}), strings.ToLower(resp["model_name"].(string))
-		rName := rData["name"].(string)
+		_, rModelName := resp, ""
+		rName := resp["name"].(string)
 		objURL := fmt.Sprintf("https://localhost/api/%s/%s-%s-%s#%s", rModelName, rModelName, rName, RANDOMUUID, rName)
```

Comment:
With line no 928, rModelName will be empty so objURL will be "https://localhost/api//-rname-uuid#rname". will this be ok if not for virtual service or vsvip?